finality-aleph/src/sync/mock/backend.rs

```diff
@@ -178,22 +183,8 @@ impl Finalizer<MockJustification> for Backend {
             None => panic!("finalizing block without a parent: {:?}", header),
         };
 
-        let parent_block = match storage.blockchain.get(&parent_id) {
-            Some(block) => block,
-            None => panic!("finalizing block without an imported parent: {:?}", header),
-        };
-
-        if parent_block.justification.is_none() {
-            panic!("finalizing block without a finalized parent: {:?}", header);
-        }
-
-        if &parent_id != storage.finalized.last().expect("there is a top finalized") {
-            panic!(
-                "finalizing block whose parent is not top finalized: {:?}. Top is {:?}",
-                header,
-                storage.finalized.last().expect("there is a top finalized")
-            );
-        }
+        // check if finalizing block without an imported parent
```

Comment:
You get much clearer errors when this prints something more than "assert failed", though I think `assert!` supports such printing without an `if`.

Reply:
fixed

Reply:
Actually doesn't seem to be?

Reply:
w8 for the push plz